docs/wiki/end-user-guide/installation/install-client.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Client
+---
+
+# Client
+
+## How to Install Cleanroom Loader on the Client
+
+### About Launchers
+
+The recommended way to run Cleanroom Loader is with a **MultiMC-based launcher**, such as **[Prism Launcher](https://prismlauncher.org/download/windows/)** or **[PolyMC](https://polymc.org/download/)**.  
+These are officially supported because other launchers may not properly handle the removal of certain vanilla libraries.
+
+### Manual Install
+
+::: info Note {id="note"}
+Importing Cleanroom Loader instance is only supported on MultiMC-based launchers. For other launchers, use the installer jar.
+:::
+Steps:
+1. Add [Fugue](https://www.curseforge.com/minecraft/mc-mods/fugue) to your mod folder  
+   *(it will not load in a Forge environment)*
+2. Add [Scalar Legacy](https://www.curseforge.com/minecraft/mc-mods/scalar-legacy) to your mod folder  
+   *(it will not load in a Forge environment)*
+3. :warning: **Warning:** Make sure your instance does not include problematic or deprecated mods.  
+   See [Preparing your modpack](/wiki/End-user-Guide/preparing-your-modpack) for more details.
+4. Download the `Cleanroom-MMC-instance-` archive from the [Releases page](https://github.com/CleanroomMC/Cleanroom/releases).
+5. Import it into your launcher.  
+   *(Alternatively, unzip the patches and JSON files into an existing 1.12 instance.)*
+6. In your launcher, select the `javaw.exe` path of your **Java 25 installation**. (`javaw.exe` will not produce any commandline output and it the one that should be used in normal cases, while `java.exe` will produce  commandline output)
+7. *(Optional)* Add JVM arguments as described in the [JVM Arguments](/wiki/End-user-Guide/args) section.
+
+### Relauncher
+
+**Relauncher** is a <ins>**Client-side**</ins> mod that allows you to start a Forge 1.12.2 instance and seamlessly switch to Cleanroom Loader.  
```

Comment:
the name of the mod is "Cleanroom Relauncher", not just "Relauncher", usage should reflect the full name